kubernetes/redpanda/templates/configmap.yaml

```diff
@@ -4,54 +4,54 @@ metadata:
+        port: {{ .Values.config.redpanda.advertised_kafka_api.port }}
       kafka_api_tls:
-        cert_file: ""
-        enabled: false
-        key_file: ""
-        truststore_file: ""
-      node_id: 0
+        cert_file: {{ .Values.config.redpanda.kafka_api_tls.cert_file }}
+        enabled: {{ .Values.config.redpanda.kafka_api_tls.enabled }}
+        key_file: {{ .Values.config.redpanda.kafka_api_tls.key_file }}
+        truststore_file: {{ .Values.config.redpanda.kafka_api_tls.truststore_file }}
+      node_id: {{ .Values.config.redpanda.node_id }}
       rpc_server:
-        address: 0.0.0.0
-        port: {{ .Values.redpanda.advertised_rpc_api.port }}
+        address: {{ .Values.config.redpanda.rpc_server.address }}
+        port: {{ .Values.config.redpanda.rpc_server.port }}
       seed_servers:
         {{- $envAll := . }}
         {{- range untilStep 0 (.Values.replicaCount|int) 1 }}
-        - node_id: {{ . }} 
+        - node_id: {{ . }}
           host:
             address: {{ template "redpanda.name" $envAll }}-{{ . }}
-            port: {{ $envAll.Values.redpanda.advertised_rpc_api.port }}
+            port: {{ $envAll.Values.config.redpanda.advertised_rpc_api.port }}
         {{- end }}
     rpk:
-      coredump_dir: /var/lib/redpanda/coredump
-      enable_memory_locking: false
-      enable_usage_stats: true
-      overprovisioned: false
+      coredump_dir: {{ .Values.config.rpk.coredump_dir }}
+      enable_memory_locking: {{ .Values.config.rpk.enable_memory_locking }}
+      enable_usage_stats: {{ .Values.config.rpk.enable_usage_stats }}
+      overprovisioned: {{ .Values.config.rpk.overprovisioned }}
       tls:
-        cert_file: ""
-        key_file: ""
-        truststore_file: ""
-      tune_aio_events: false
-      tune_clocksource: false
-      tune_coredump: false
-      tune_cpu: false
-      tune_disk_irq: false
-      tune_disk_nomerges: false
-      tune_disk_scheduler: false
-      tune_fstrim: false
-      tune_network: false
-      tune_swappiness: false
-      tune_transparent_hugepages: false
+        cert_file: {{ .Values.config.rpk.tls.cert_file }}
+        key_file: {{ .Values.config.rpk.tls.key_file }}
+        truststore_file: {{ .Values.config.rpk.tls.truststore_file }}
+      tune_aio_events: {{ .Values.config.rpk.tune_aio_events }}
+      tune_clocksource: {{ .Values.config.rpk.tune_clocksource }}
+      tune_coredump: {{ .Values.config.rpk.tune_coredump }}
+      tune_cpu: {{ .Values.config.rpk.tune_cpu }}
+      tune_disk_irq: {{ .Values.config.rpk.tune_disk_irq }}
+      tune_disk_nomerges: {{ .Values.config.rpk.tune_disk_nomerges }}
+      tune_disk_scheduler: {{ .Values.config.rpk.tune_disk_scheduler }}
+      tune_fstrim: {{ .Values.config.rpk.tune_fstrim }}
+      tune_network: {{ .Values.config.rpk.tune_network }}
+      tune_swappiness: {{ .Values.config.rpk.tune_swappiness }}
+      tune_transparent_hugepages: {{ .Values.config.rpk.tune_transparent_hugepages }}
```

Comment:
Life could be easier:
`{{ toYaml .Values.config | indent 4}}`

Reply:
Oh, I do see some stuff that isn't 1:1.

Reply:
Yes, thanks, `toYaml` helps. I still need to range over the seed servers though.